index.html

```diff
@@ -14,41 +15,139 @@
     />
 
   -->
-    <link rel="stylesheet" href="style.css" />
-
-    <script
-      src="https://kit.fontawesome.com/26c38390b4.js"
-      crossorigin="anonymous"
-    ></script>
-  </head>
-  <body>
-    <header class="navbar">
-      <a href="#" class="brand-title">My logo</a>
-      <i class="fas fa-bars"></i>
-    </header>
+  <link rel="stylesheet" href="style.css" />
+
+  <script src="https://kit.fontawesome.com/26c38390b4.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+
+  <header class="navbar">
+    <a href="#" class="brand-title">My logo</a>
+    <i class="fas fa-bars"></i>
+  </header>
+
+  <container class="page">
 
     <section class="presentation">
-      <h2>I’m Alexandra. Glad to see you!</h2>
+      <h2>I’m Gonzalo. Glad to see you!</h2>
+
+      <article class="job">
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+      </article>
+      <section class="contact">
+        <p>LET’S CONNECT</p>
+      </section>
+      <section class="socialmedia">
+        <ul>
+          <li><i class="fa-brands fa-twitter"></i></li>
+          <li><i class="fa-brands fa-linkedin"></i></li>
+          <li><i class="fa-solid fa-m"></i></li>
+          <li><i class="fa-brands fa-square-github"></i></li>
+          <li><i class="fa-regular fa-hand-peace"></i></li>
+        </ul>
+      </section>
     </section>
 
-    <article class="job">
-      <p>
-        I’m a software developer! I can help you build a product , feature or
-        website Look through some of my work and experience! If you like what
-        you see and have a project you need coded, don’t hesitate to contact me.
-      </p>
-    </article>
-    <section class="contact">
-      <p>LET’S CONNECT</p>
+    <section class="cards">
+
+      <div class="card">
+
+        <img class="cardimg" src="./images/tonic.png" alt="">
+
+        <div class="grid-item-title-1">
+          <p class="title">Tonic</p>
+          <div class="canopy">
+            <p>CANOPY</p>
+            <ul>
+              <li> Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <article>A daily selection of privately personalized reads;
+            no accounts or sign-ups required.</article>
+        </div>
+
```

Comment:
- [ ] It is great to add some item in your portfolio 👍 , however as describe in the Figma template you should have at 4 items to stick to the Figma template and meet the requirements 👍 , Kindly fix that

- [ ] KIndly make sure these text has a correct `font-family` as describe in the Figma template 👍 

- [ ] Make sure the Item is well centered in the page 👍 

![Capture d’écran 2023-03-29 à 2 47 32 AM](https://user-images.githubusercontent.com/80458199/228401383-9ff39158-5b3f-4beb-b74b-84a0be7e98aa.png)